map.js

```diff
@@ -0,0 +1,80 @@
+    function showOrHideMap() {
+        if (document.getElementById("map").style.display == 'none') {
+            document.getElementById("map").style.display = 'block';
+        }else {
+            document.getElementById("map").style.display = 'none';
+        }
+    }
+
+    function initAutocomplete() {
+    var map = new google.maps.Map(document.getElementById('map'), {
+        center: {lat: -27.5762162, lng: -48.532195},
+        zoom: 13,
+        mapTypeId: google.maps.MapTypeId.ROADMAP
+    });
+
+    // Create the search box and link it to the UI element.
+    var input = document.getElementById('pac-input');
+    var searchBox = new google.maps.places.SearchBox(input);
+    map.controls[google.maps.ControlPosition.TOP_LEFT].push(input);
+
+    // Bias the SearchBox results towards current map's viewport.
+    map.addListener('bounds_changed', function() {
+        searchBox.setBounds(map.getBounds());
+    });
+
+    var markers = [];
+    // Listen for the event fired when the user selects a prediction and retrieve
+    // more details for that place.
+    searchBox.addListener('places_changed', function() {
+        var places = searchBox.getPlaces();
+
+        if (places.length == 0) {
+            return;
+        }
+
+        // Clear out the old markers.
+        markers.forEach(function(marker) {
+            marker.setMap(null);
+        });
+        markers = [];
+
+        // For each place, get the icon, name and location.
+        var bounds = new google.maps.LatLngBounds();
+        places.forEach(function(place) {
+            var icon = {
+                url: place.icon,
+                size: new google.maps.Size(71, 71),
+                origin: new google.maps.Point(0, 0),
+                anchor: new google.maps.Point(17, 34),
+                scaledSize: new google.maps.Size(25, 25)
+            };
+
+            // Create a marker for each place.
+            markers.push(new google.maps.Marker({
+                map: map,
+                icon: icon,
+                title: place.name,
+                position: place.geometry.location
+            }));
+
+            // HERE IS THE KEY TO PARADISE - START

```

Comment:
Sorry, this was my last piece of code and I was pretty happy... and tired :)
I am aware in real life this is not acceptable. Already changed to "//
Create a click listener for each marker"

2016-03-02 17:22 GMT-03:00 André Luiz Cardoso <notifications@github.com>:

> In map.js
> <https://github.com/otaviofontana/arcTouchOtavioFontana/pull/1#discussion_r54784811>
> :
>
> > +                url: place.icon,
> > +                size: new google.maps.Size(71, 71),
> > +                origin: new google.maps.Point(0, 0),
> > +                anchor: new google.maps.Point(17, 34),
> > +                scaledSize: new google.maps.Size(25, 25)
> > +            };
> > +
> > +            // Create a marker for each place.
> > +            markers.push(new google.maps.Marker({
> > +                map: map,
> > +                icon: icon,
> > +                title: place.name,
> > +                position: place.geometry.location
> > +            }));
> > +
> > +            // HERE IS THE KEY TO PARADISE - START
>
> What is the key to paradise?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/otaviofontana/arcTouchOtavioFontana/pull/1/files#r54784811>
> .
>